sharepoint/sharepoint-ps/sharepoint-online/Get-SPOInformationBarriersInsightsReport.md

```diff
@@ -0,0 +1,163 @@
+﻿---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spoinformationbarriersinsightsreport
+applicable: SharePoint Online
+title: Get-SPOInformationBarriersInsightsReport
+schema: 2.0.0
+author: pvrk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Get-SPOInformationBarriersInsightsReport
+
+## SYNOPSIS
+
+This cmdlet helps to view the status of the insights on Information Barrier (IB).
+
+## SYNTAX
+
+```powershell
+Get-SPOInformationBarriersInsightsReport [-ReportId <Guid>] [-Section <SectionType>] [-Action <ActionType>] [-Service <ServiceType>] [-FullDetails] [<CommonParameters>]
```

Comment:
>[-FullDetails] [](http://example.com/codeflow?start=133&length=14)

is there a backend change that's adding this new param? as I don't see it in current cmdlet